<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>551</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>World Clock</string>
     </attribute>
     <widget class="QPushButton" name="btn_ir_worldclock">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MRT_Nil 3</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_worldclock">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0:0:0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_de_worldclock">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MRT_Nil 3</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_us_worldclock">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MRT_Nil 3</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Alarm</string>
     </attribute>
     <widget class="QLineEdit" name="le_name_alarm">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_time_alarm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>time</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_description__alarm">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>90</y>
        <width>161</width>
        <height>180</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_alarm">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>241</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Stopwatch</string>
     </attribute>
     <widget class="QLabel" name="label_stopwatch">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0:0:0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_start_stopwatch">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stop_stopwatch">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset_stopwatch">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Timer</string>
     </attribute>
     <widget class="QLineEdit" name="tb_h">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>50</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="tb_m">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>15</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="tb_s">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>30</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stop_timer">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_start_timer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>50</y>
        <width>15</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>317</x>
        <y>50</y>
        <width>15</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
